_episodes/03-index-slice-subset.md

```diff
@@ -358,22 +358,28 @@ gives the **output**
 Remember that Python indexing begins at 0. So, the index location [2, 6]
 selects the element that is 3 rows down and 7 columns over in the DataFrame.
 
+It is worth noting that rows are selected when using `loc` with a single list of labels (or `iloc` with a single list of integers). However, unlike `loc` or `iloc`, indexing a data frame directly with labels will select columns, while ranges of integers will select rows. Direct indexing of rows is redundant with using `iloc`, and will raise a `KeyError` if a single integer or list is used; the error will also occur if index labels are used without `loc` (or column labels used with it).
```

Comment:
This paragraph provides a useful summary but I found it quite difficult to interpret the first couple of times I read it through. I think it could be made clearer with the inclusion of an example for each case, e.g. 

> However, unlike `loc` or `iloc`, indexing a data frame directly with labels (e.g. `surveys_df['species_id', 'plot_id', 'weight']`) will select columns, while ranges of integers (e.g. `surveys_df[0:13]`) will select rows.